cityofaustin/atd-moped
# Makes use of phase <> subphase db relationship

## Associated issues
https://github.com/cityofaustin/atd-data-tech/issues/9732

## Testing
**URL to test:** Moped test instance or local

**Steps to test:**
Navigate to a project's Timeline tab. Create a few phases with subphases. Verify subphases appear in dropdown and save correctly.

E.g., select a "potential" phase - your subphase options should be:

- Study recommended
- Study in progress
- Recommended
- Environmental study
- Active development review

For a "planned" phase, your subphase options should be:

- Environmental study
- Study in progress
- Ready for design
-

---
#### Ship list
